### DISCUSSION TITLE
开启G-SYNC后是否还需要打开video-sync=display-xxxx？

### DISCUSSION THREAD

**Discussion by ptr7F** [hooke007/mpv_PlayKit#259] [Q&A 问答] [answered]
默认的NVIDIA控制面板是开启G-SYNC的，然后我配置为video-sync=display-resample后，看display-stats，显示为
![image](https://user-images.githubusercontent.com/43652659/214058658-740a3846-c055-44c3-9d47-b81844e45356.png)
Display FPS，estimated的帧数为当前显示器刷新率的两倍，并且用显示器自带的FPS计算器显示有144。关闭G-SYNC后，estimated为当前显示器刷新率。

不使用video-sync=display-resample后，在开启G-SYNC的情况下，播放视频似乎已经在G-SYNC的使用条件下了，播放30帧的视频，显示器自带的FPS计算大约是60左右，但FPS计算一直在跳动，不是恒定的60FPS。

请问开启G-SYNC是不是没必要开启video-sync=display-xxxx了？

**Comment by hooke007** [comment, marked as answer]
你理解错误。不兼容的意思是——关闭gsync

**Reply by ptr7F** [reply]
意思是G-SYNC和video-sync=display-xxxx只能二选一？

**Reply by hooke007** [reply]
首先，gsync本身有工作范围限制，非ultimate版48Hz~你的显示器最高刷新率，ultimate版能下探到1Hz。
非ultimate版无法作用到23/24Hz。
目前display-resample直接对齐到显示器最高刷新率，而且工作逻辑和gsync冲突。

其次，单独禁用而不是全局禁用，不影响你打游戏用gsync

**Reply by ptr7F** [reply]
ok，了解，谢谢！

**Reply by hooke007** [reply]
这些内容在faq已有记录

**Reply by hooke007** [reply]
另外你也可以使用gsync而不是video-sync。
不推荐的理由是极其影响界面的操作流畅度，以及可能引入其他未知的问题。

**Reply by ptr7F** [reply]
好的，谢谢解答！
